Cleanup and restructure API reference overview (#140)
canonical/anbox-cloud-docs: reference/api-reference.md

(ref-api)=
# API reference

> Anbox Cloud includes various APIs to help you building solutions with it. This page provides an overview over the available APIs and what they are used for.

Reviewer: ```suggestion
Anbox Cloud includes various APIs to help you build solutions with it. This page provides an overview of the available APIs and their uses.
```

All these APIs except for the {ref}`anbox-https-api` have an auto-generated Open API specification describing its API endpoints.

## Available APIs

### AMS HTTP API

> The AMS services provide a HTTP based API to allow you managing all of its objects. Internally the `amc` client uses the same API to perform its operation.

Reviewer: vowel sound, so an HTTP
```suggestion
The AMS services provide an HTTP based API to help you manage all of its objects. Internally the `amc` client uses the same API to perform its operation.
```

You can find the OpenAPI specification [here](https://canonical.github.io/anbox-cloud.github.com/latest/ams/).

### Anbox HTTP API

> The Anbox runtime inside each instance managed by AMS provides a HTTP API over a UNIX domain socket to allow interacting with various functionality Anbox provides. This for example includes the injection of position or sensor events into Android.

Reviewer: ```suggestion
The Anbox runtime inside each instance managed by the AMS provides an HTTP API over a UNIX domain socket that helps interacting with various functionalities offered by Anbox. This includes the injection of position or sensor events into Android, among other things.
```

You can find the API specifiction at {ref}`anbox-https-api`.

### Anbox Stream Gateway API

> The Anbox Stream Gateway service provides an API endpoint to allow you managing streaming sessions.

Reviewer: ```suggestion
The Anbox Stream Gateway service provides an API endpoint to help you to manage streaming sessions.
```
